README.md

```diff
@@ -0,0 +1,5 @@
+[bsc#999203](https://bugzilla.suse.com/show_bug.cgi?id=999203)
+
+```console
+$ ruby print_deps.rb YaST:Head openSUSE_Factory x86_64 yast_deps.yaml
+```
```

Comment:
Improved. Regarding openSUSE:Factory, let's change that once it is ready, like with the cache dir placement.